documentation/how-to/observability.md

```diff
@@ -0,0 +1,129 @@
+in order to collect logs and telemetry.
+
+Enabling monitoring requires that you:
+
+- Have an active [Charmed Temporal K8s operator](../tutorial/00-introduction.md)
+  deployed.
+
+## Deploy COS Lite Bundle
+
+To enable monitoring, we will be deploying cos-lite to a separate model and
+exposing the necessary endpoints as offers, relating them to the Temporal server
+using cross-model relations. To get started, we must first deploy the COS Lite
+bundle:
+
+```bash
+juju add-model cos
+juju deploy cos-lite --trust
+```
+
+Wait until the applications are ready - when they are ready, `juju status` will
+show:
+
+```
+Model  Controller           Cloud/Region        Version  SLA          Timestamp
+cos    temporal-controller  microk8s/localhost  3.1.5    unsupported  11:10:19+03:00
+
+App           Version  Status   Scale  Charm             Channel  Rev  Address         Exposed  Message
+alertmanager  0.25.0   active       1  alertmanager-k8s  stable    76  10.152.183.185  no
+catalogue              active       1  catalogue-k8s     stable    13  10.152.183.176  no
+grafana       9.2.1    active       1  grafana-k8s       stable    81  10.152.183.78   no
+loki          2.7.4    active       1  loki-k8s          stable    89  10.152.183.236  no
+prometheus    2.33.5   active       1  prometheus-k8s    stable   103  10.152.183.69   no
+traefik       2.9.6    waiting      1  traefik-k8s       stable   110  10.152.183.223  no       installing agent
+
+Unit             Workload  Agent  Address       Ports  Message
+alertmanager/0*  active    idle   10.1.232.120
+catalogue/0*     active    idle   10.1.232.108
+grafana/0*       active    idle   10.1.232.123
+loki/0*          active    idle   10.1.232.121
+prometheus/0*    active    idle   10.1.232.122
+traefik/0*       waiting   idle   10.1.232.119         gateway address unavailable
+```
+
+Run the following commands to offer COS interfaces to be cross-model related
+with the Charmed Temporal K8s model:
+
+```bash
+# Expose the cos integration endpoints
+juju offer prometheus:metrics-endpoint
+juju offer loki:logging
+juju offer grafana:grafana-dashboard
+```
+
+Once done, the output of `juju status` should be updated to show the following
+offers:
+
+```
+Offer       Application  Charm           Rev  Connected  Endpoint           Interface          Role
+grafana     grafana      grafana-k8s     81   0/0        grafana-dashboard  grafana_dashboard  requirer
+loki        loki         loki-k8s        89   0/0        logging            loki_push_api      provider
+prometheus  prometheus   prometheus-k8s  103  0/0        metrics-endpoint   prometheus_scrape  requirer
+```
+
+## Relate Temporal Server to COS Lite Bundle
+
+```bash
+# Relate Temporal to the cos-lite apps
+juju switch temporal-model
+juju relate temporal-k8s admin/cos.grafana
+juju relate temporal-k8s admin/cos.loki
+juju relate temporal-k8s admin/cos.prometheus
+```
+
+Once done, the output of `juju status` should be as follows:
```

Comment:
`juju status --relations` could be more appropriate here

Reply:
Thanks for the thorough review and feedback Amber! I've updated the docs accordingly.